pg_search/src/api/search.rs

```diff
@@ -104,3 +108,37 @@ fn drop_bm25_internal(index_name: &str) {
     SearchIndex::drop_index(&writer_client, index_name)
         .unwrap_or_else(|err| panic!("error dropping index {index_name}: {err}"));
 }
+
+#[pg_extern]
+#[allow(unused)]
```

Comment:
Mostly a Q out of curiosity - do you need to specify `#[allow(unused)]` if the function is being exposed via `#[pg_extern]`?

Reply:
No you don't! This shouldn't be here, thank you for catching.